hawkbit-rest/hawkbit-ddi-resource/src/test/java/org/eclipse/hawkbit/ddi/rest/resource/DdiRootControllerTest.java

```diff
@@ -448,8 +442,14 @@ private void assertThatAttributesUpdateIsNotRequested(final String targetControl
                 .andExpect(jsonPath("$._links.configData").doesNotExist());
     }
 
-    private ResultActions sendDeploymentActionFeedback(final Target target, final Action action, final String feedback)
-            throws Exception {
+    private ResultActions sendDeploymentActionFeedback(final Target target, final Action action, final String execution,
+            String finished, String message) throws Exception {
```

Comment:
Most tests don't use the `message` param. May I ask you to add another `sendDeploymentActionFeedback(...)` method without mwssage param and forward internally to this one with `null`?